src/components/common/VCheckboxRemember.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <v-checkbox
+    :model-value="value"
```

Comment:
vmodel?

Reply:
we have `value` prop, v-model does not work correctly